docs/stable/14-mentions.md

```diff
@@ -0,0 +1,27 @@
+---
+permalink: /specs/14
+parent: Stable specs
+title: 14/MENTIONS
+---
+
+# 14/MENTIONS
+
+## Usage
+
+To mention a user, one would type a message containing `@0xpk` where `pk` is the public key of the [user account](https://specs.status.im/spec/2) to be mentioned. The message is sent as a [regular message with `TEXT_PLAIN` content type](https://specs.status.im/spec/6#content-types).
+For better user experience, the client should display a known [ens name or the 3-word pseudonym corresponding to the key](https://specs.status.im/spec/2#contact-verification) instead of the `pk`.
+
+## Implementation
+
+`status-go` parses the markdown in the messages and detects mentions that follow the pattern described above. This is done in [`parser.inline.go`](https://github.com/status-im/markdown/blob/b9fe921681227b1dace4b56364e15edb3b698308/parser/inline.go#L68). When mentions are found they are added as an element to the message that is passed to the UI with the following format:
+
+```
+{:type "mention" :literal "0x......"}
+```
+
+`status-go` currently parses the following markdown elements: `code`, `emph`, `strong`, `link`, `status-tag` and `mention`.
+`mentions` should be displayed in a color that sets it apart from regular text and tapping it should open the chat with the mentioned user.
```

Comment:
Is there a dedicated section to markdown? Are the other markdown elements documented?

Reply:
```suggestion
`status-go` currently parses the following markdown elements: `code`, `emph`, `strong`, `link`, `status-tag` and `mention`.
A `mention` tag SHOULD be displayed in a color that sets it apart from regular text and tapping it should open the chat with the mentioned user.
```